controllers/rest/operation_initializers.go

```diff
@@ -225,6 +226,22 @@ func UserDefinedInitializer(ctxt context.Context, api *RESTAPI, path string, met
 		ctxt = context.WithValue(ctxt, weoscontext.EVENT_STORE, eventStore)
 	}
 
+	for _, resp := range operation.Responses {
```

Comment:
I think this will be better placed in the RouteInitializer. My thinking is that it's in this initializer that we actually setup the routes on the router (which is what is essentially happening when you call api.e.Static)